packages/compass-deployment-awareness/src/components/unknown.module.less

```diff
@@ -1,4 +1,4 @@
-@import (reference) "~less/compass/_theme.less";
+@import (reference) "mongodb-compass/src/app/styles/index.less";
```

Comment:
Isn't this a circular dep? Wouldn't it create issues?

Reply:
This creates a circular dependency indeed, but this should not create issues as this dependency is needed more or less only in dev mode (either for building the plugins or running the playground). I also double-checked that cloud and charts are not using any of the files with those deps, so it won't be an issue for them. This _could be_ a problem for lerna, but this is why I'm avoiding adding this dependency to the `devDependencies` list, ignoring it in the depcheck instead. As I mentioned in the PR description:

> This change also required to hackily refer to mongodb-compass as a library in a few places creating a recursive dependency, but imo this clean up is worth it and eventually we can move all things that are actually shared between plugins and compass in their own, separate place

So it is a hack, but I think we will be able to get away from this before it will cause any trouble